client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx

```diff
@@ -92,6 +98,34 @@ const AddSubscribersModal = ( { site }: AddSubscribersModalProps ) => {
 				</>
 			) }
 
+			{ ! isUploading && isImportInProgress && ! hasStaleImportJobs && (
+				<Notice
+					className="add-subscribers-modal__notice"
+					isCompact
+					isReskinned
+					status="is-info"
+					showDismiss={ false }
+				>
+					{ translate(
+						'Your subscribers are being imported. This may take a few minutes. You can close this window and we’ll notify you when the import is complete.'
+					) }
+				</Notice>
+			) }
+
+			{ ! isUploading && isImportInProgress && hasStaleImportJobs && (
+				<Notice
+					className="add-subscribers-modal__notice"
+					isCompact
+					isReskinned
+					status="is-warning"
+					showDismiss={ false }
+				>
+					{ translate(
+						'Your recent import is taking longer than expected to complete. If this issue persists, please contact our support team for assistance.'
+					) }
```

Comment:
The copy is clear and looking good to me. 👍🏼 Maybe we could also consider including a link to the support page, e.g. https://wordpress.com/support/help-support-options/. 🙂

Reply:
Thank you for the feedback @ivan-ottinger! Do you think the link should be the same for atomic and jetpack sites as well?

Reply:
Good point. I think for Jetpack sites we could use https://jetpack.com/contact-support/ - as it is being used in other places in the Calypso codebase and caters information for both free and paid plan users.

Reply:
Sounds good, I'll include this one as well. Thank you!